packages/core/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "rootDir": "./src",
     "strict": true,
     "sourceMap": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "esModuleInterop": true
```

Comment:
This shit did breaks things for me recently, so I removed it, what this is for?

Reply:
I mean `esModuleInterop`

Reply:
To allow just import not require, ok let me fix it as it was.